src/components/SupportedAddOns.js

```diff
@@ -189,7 +189,7 @@ class SupportedAddOns extends React.Component {
             </div>
           </div>
           <div className="flex justifyContent--flexEnd">
-            <a href={`https://github.com/replicatedhq/kURL/tree/master/addons/${addOn.name}`} target="_blank" rel="noopener noreferrer"> <i className="icon u-externalLinkIcon clickable"></i> </a>
+            <a href={`https://github.com/replicatedhq/kURL/tree/master/addons/${(addOn.name === "openEBS") ? 'openebs' : addOn.name}`} target="_blank" rel="noopener noreferrer"> <i className="icon u-externalLinkIcon clickable"></i> </a>
```

Comment:
why not something like `addOn.name.toLowerCase()`?
```suggestion
            <a href={`https://github.com/replicatedhq/kURL/tree/master/addons/${addOn.name.toLowerCase()}`} target="_blank" rel="noopener noreferrer"> <i className="icon u-externalLinkIcon clickable"></i> </a>
```